packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/components/MenuOption.tsx

```diff
@@ -15,7 +15,7 @@ interface Props {
 }
 
 export const MenuOption: FunctionComponent<Props> = ({ children, className, formatting }) => {
-    if (formatting === 'multiple') {
+    if (formatting === 'multiple' || formatting === 'block-quote') {
```

Comment:
What is this change for? It removes the left border from the "Quote" menu item. 

This is the current PR implementation:

![image](https://user-images.githubusercontent.com/370680/167393501-41ea8f78-af37-4814-86df-6628371e728a.png)


From what I see, it should have the border, if the designs are up-to-date. This is from [Figma](https://www.figma.com/file/KD30VWNjFh6mVWkg6Taqhe/06-%E2%80%93-Editor?node-id=1869%3A3584):

![image](https://user-images.githubusercontent.com/370680/167393556-6cdd0d7e-e63c-47a9-8608-22fb5a8d78e6.png)

Reply:
![Screenshot 2022-05-12 at 09 11 08](https://user-images.githubusercontent.com/41072665/168028808-7935e7f6-3723-4066-841c-9c5d0529b87a.png)
I've did this because of problem with styling in dropdown. I've realized there is a place for styling dropdown elements. Fixed